src/site/_includes/pattern-set.njk

```diff
@@ -0,0 +1,132 @@
+---
+layout: default
+pageScripts:
+  - '/js/content.js'
+---
+
+{% if page.url %}
+  {% set patternSetId = page.url | getRelativePath('/patterns/') %}
+{% else %}
+  {% set patternSetId = '' %}
+{% endif %}
+
+{% if patternSetId %}
+  {% set patternSet = patterns.sets[patternSetId] %}
+{% endif %}
+
+{% if patternSet.suite %}
+  {% set patternSuite = patterns.sets[patternSet.suite] %}
+  {% set currentPatternSet = patternSet %}
+{% else %}
+  {% set patternSuite = patterns.sets[patternSetId] %}
+{% endif %}
+
+<section class="w-layout-container">
+  {% from 'macros/breadcrumbs.njk' import breadcrumbs with context %}
+  {% set crumbs = [
+    ['All patterns', '/patterns/'],
+    [patternSuite.title, '/patterns/' + patternSuite.id]
+  ] %}
+  {{ breadcrumbs(crumbs) }}
+</section>
+
+<header class="w-masthead-about">
+  <div class="w-masthead-about__container">
```

Comment:
Hm... it's not great that we're reusing this component (it's kind of coupled to the about page) but I understand why it's being used.

I'd suggest overriding it a bit and setting padding top/bottom to 0 and replacing them with margin top/bottom 64px.

Otherwise there will be more whitespace above the divider than below it.